src/style/_mixins.scss

```diff
@@ -1,5 +1,14 @@
 @import "./shadow.mixin";
 
+$ssv-breakpoints: (
+	"xsmall": 450px,
+	"small": 767px,
+	"medium": 992px,
+	"large": 1200px,
+	"xlarge": 1500px,
+	"xxlarge": 1800px
+);
```

Comment:
agreed, it would be nice; this needs to be done something as following tho not with default

```scss
$ssv-breakpoints: () !default;
$ssv-breakpoints-defaults: (
	"xsmall": 450px,
	"small": 767px,
	"medium": 992px,
	"large": 1200px,
	"xlarge": 1500px,
	"xxlarge": 1800px
);
$ssv-breakpoints: map-merge($ssv-breakpoints-defaults, $ssv-breakpoints);
```